test/regression/expected/domain.out

```diff
@@ -0,0 +1,109 @@
+create domain domainvarchar varchar(5) check (value is not null);
+create domain domainnumeric numeric(8,2)  check (value is not null);
+create domain domainint4 int4 check (value > 0);
+create domain domaintext text check (value is not null);
+-- Test tables using domains
+create table basictest
+           ( testint4 domainint4
+           , testtext domaintext
+           , testvarchar domainvarchar
+           , testnumeric domainnumeric
+           );
+INSERT INTO basictest values ('88', 'haha', 'short', '123.12');      -- Good
+INSERT INTO basictest values ('88', 'haha', 'short text', '123.12'); -- Bad varchar
+ERROR:  value too long for type character varying(5)
+INSERT INTO basictest values ('88', 'haha', 'short', '123.1212');    -- Truncate numeric
+-- domain check
+INSERT INTO basictest values ('-1', 'haha', 'short', '123.1212');   -- Bad int4
+ERROR:  value for domain domainint4 violates check constraint "domainint4_check"
+INSERT INTO basictest values ('88', NULL, 'short', '123.1212');    -- Bad text
+ERROR:  value for domain domaintext violates check constraint "domaintext_check"
+INSERT INTO basictest values ('88', 'haha', NULL, '123.1212');   -- Bad varchar
+ERROR:  value for domain domainvarchar violates check constraint "domainvarchar_check"
+INSERT INTO basictest values ('88', 'haha', 'short', NULL);    -- Bad numeric
+ERROR:  value for domain domainnumeric violates check constraint "domainnumeric_check"
+SELECT 5::domainint4; -- Good
+ domainint4 
+------------
+          5
+(1 row)
+
+SELECT (-5)::domainint4; -- Bad int4
```

Comment:
This also doesn't go through duckdb execution, because there is no FROM clause (or duckdb-only function). So let's just remove it.